libpod/runtime_ctr.go

```diff
@@ -436,7 +437,7 @@ func (r *Runtime) LookupContainer(idOrName string) (*Container, error) {
 // Filters can be provided which will determine what containers are included in
 // the output. Multiple filters are handled by ANDing their output, so only
 // containers matching all filters are returned
-func (r *Runtime) GetContainers(filters ...ContainerFilter) ([]*Container, error) {
+func (r *Runtime) GetContainers(filters ...filterable.Filter) ([]*Container, error) {
```

Comment:
I think we can keep the definition for ContainerFilter in libpod - anything that matches the signature will be treated by Go as a ContainerFilter, even if it's defined in another package

Reply:
Oh, I see you redefined it. Hmmm.

Can we have filterable.Filter produce a ContainerFilter instead? I kind of like arbitrary-function interface we have going on here, and I'd like to keep it if we're retaining filtering.